client/extensions/woocommerce/state/data-layer/orders/send-invoice/index.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { createNoteSuccess } from 'woocommerce/state/sites/orders/notes/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import {
+	orderInvoiceFailure,
+	orderInvoiceSuccess,
+} from 'woocommerce/state/sites/orders/send-invoice/actions';
+import request from 'woocommerce/state/sites/http-request';
+import { WOOCOMMERCE_ORDER_INVOICE_SEND } from 'woocommerce/state/action-types';
+
+export const fetch = action => {
```

Comment:
Send might make more sense then `fetch` here.

Reply:
Though I agree, `fetch` is the property expected by `dispatchRequestEx`, and shouldn't be used outside of this file & the test, so I think I'm going to leave it as-is.